### PR TITLE
docker image for migration script

### PR DESCRIPTION
**Pull request by burdettadam**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Setup Image Metadata
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            ghcr.io/${{ steps.info.outputs.repo-owner }}/acapy-wallet-upgrade
```
Comment (dbluhm) [inline .github/workflows/publish.yml]: To match the future project name, perhaps we should do:
```suggestion
            ghcr.io/${{ steps.info.outputs.repo-owner }}/aries-acapy-wallet-upgrade
```

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+# Setup project
+COPY pyproject.toml poetry.lock README.md ./
+COPY acapy_wallet_upgrade/ acapy_wallet_upgrade/
+COPY tests/ tests/
+RUN poetry build
+
+
+FROM --platform=linux/amd64 python:3.10-slim AS askar-upgrade
+COPY --from=base /usr/src/app/dist/acapy_wallet_upgrade-0.1.0-py3-none-any.whl /tmp/acapy_wallet_upgrade-0.1.0-py3-none-any.whl
```
Comment (dbluhm) [inline docker/Dockerfile]: Can we use a wildcard to select the wheel file so we don't have to modify the dockerfile when the version changes?

```suggestion
COPY --from=base /usr/src/app/dist/acapy_wallet_upgrade-*-py3-none-any.whl /tmp/.
```

Something like this perhaps?

File: docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+RUN pip install /tmp/acapy_wallet_upgrade-0.1.0-py3-none-any.whl && \
```
Comment (dbluhm) [inline docker/Dockerfile]: Same here, if we can use a wildcard so we don't have to update versions, that would be ideal.

File: docker/README.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Running Migration Script From Docker Container
```
Comment (dbluhm) [inline docker/README.md]: What do you think of putting this in the main readme and just having the dockerfile in the root of the project? i.e. no `docker` folder?

Comment (burdettadam) [reply docker/README.md]: I prefer a docker folder, but am open to the idea of moving it.

Comment (burdettadam) [reply docker/README.md]: Collapsing Docker folder into.

**Review by dbluhm — APPROVED** [review]

**Pull request merged**